### PR TITLE
Update language setup and add ansible

### PR DESCRIPTION
**Pull request by pericnenad**
Here is a list of changes and additions done

- Guess the language by reading the directory of the new language only
- Modify Makefile for easier finding and editing of the language configuration (LANG_DIR)
- Added ansible as template language
- Changed LANG_DIR in 'python tests' to LOCAL_LANG_DIR so it conforms with the structure of the rest of the files in the directory
- Modify README to explain the LANG modification
- add simple .gitignore block for editors

### CODE REVIEW

**Comment by pericnenad** [comment]
I just realized, maybe we could actually make a "template" directory which can be used by any new language, instead of copying the "python" to everything? 
Does that make sense as an idea?

File: makefile/.konveyor/ansible/.pre-commit-config.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+repos:
```
Comment (mpryc) [inline makefile/.konveyor/ansible/.pre-commit-config.yaml:1]: Is this file required here? I mean ansible do have different then isort/flake8 targets?

File: makefile/.konveyor/ansible/tests.mk [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+LOCAL_LANG_DIR = $(notdir $(patsubst %/,%,$(dir $(LOCAL_PATH))))
+
+$(LOCAL_LANG_DIR)-tests:
+	$(MAKE) pre-commit
+
+pre-commit: setup-pre-commit
+	. ${PYTHON_VENV}/bin/activate && pre-commit
+
+isort: setup-pre-commit
+	. ${PYTHON_VENV}/bin/activate && pre-commit run isort
```
Comment (mpryc) [inline makefile/.konveyor/ansible/tests.mk:12]: is this really target for ansible based projects?

File: makefile/.konveyor/ansible/tests.mk [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+flake8: setup-pre-commit
+	. ${PYTHON_VENV}/bin/activate && pre-commit run flake8
```
Comment (mpryc) [inline makefile/.konveyor/ansible/tests.mk:15]: same as above.

File: makefile/.konveyor/ansible/setup_env.mk [hunk 4]
```diff
@@ -0,0 +1,13 @@
+### Important, the LANG_DIR must match one from Makefile
+LOCAL_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
```
Comment (mpryc) [inline makefile/.konveyor/ansible/setup_env.mk:2]: Nice ! :) we can remove the LANG_DIR comment above.

**Review by mpryc — CHANGES_REQUESTED** [review]

**Comment by pericnenad** [comment]
You are right.
I copy/pasted. I should have paid attention, was overzealous in pushing :D 
Changing now!

File: makefile/.konveyor/ansible/tests.mk (hunk 2 above)
Comment (pericnenad) [reply makefile/.konveyor/ansible/tests.mk]: Same as above, removing. Was pasted along with the dir...

File: makefile/.konveyor/ansible/tests.mk (hunk 3 above)
Comment (pericnenad) [reply makefile/.konveyor/ansible/tests.mk]: Yup yup same.

File: makefile/.konveyor/ansible/setup_env.mk (hunk 4 above)
Comment (pericnenad) [reply makefile/.konveyor/ansible/setup_env.mk:1]: Done , and expanded to include python dir as well. So it is all the same.
